src/components/Header.js

```diff
@@ -1,13 +1,27 @@
 import React, { Component } from 'react';
 
 class Header extends Component {
+	constructor(props) {
+		super(props);
+		
+		this.keyPress = this.keyPress.bind(this);
+	}
+	
+	keyPress(e) {
+		if (e.charCode == 13) {
+			this.props.addEntry(e.target.value);
+			e.target.value = '';
+		}
+	}
+	
 	render() {
 		return (
 			<header className="header">
 				<h1>todos</h1>
 				<input
 					className="new-todo"
 					placeholder="What needs to be done?"
+					onKeyPress={(e) => this.keyPress(e)}
```

Comment:
Doesn't that then require me to separately bind it to the instance?  I did it this way to avoid that.  Did I accidentally leave in the line where it's bound to the instance?